### ISSUE TITLE
RELU

### ISSUE THREAD

**Issue by huangchaoxing** [zzzyq/Depth-Map-Prediction-with-Multi-Scale-Deep-network#3]
**It seems that the output of each block doesn't go trough the activation function,while only the last layer's output should be linear,as being mentioned in the paper**
"
class fineNet(nn.Module):
    def __init__(self, init_weights=True):
        super(fineNet, self).__init__()
        self.conv1 = nn.Conv2d(3,63,kernel_size=9,stride=2,padding=1)
        self.conv2 = nn.Conv2d(64,64,kernel_size=5,stride=1,padding=1)
        self.conv3 = nn.Conv2d(64,1,kernel_size=1,stride=1,padding=1)
        if init_weights:
            self._initialize_weights()

    def forward(self, x, y):
        x = F.max_pool2d(self.conv1(x), 3, stride=2)
        x = torch.cat((x,y),1)
        x = self.conv2(x)
        x = self.conv3(x)
        return x
"

**Comment by Toobbby** [comment]
我读了一下原文，感觉除了relu的问题还有一个就是文章中提到的coarse1 这一层网络代码里面是不是没有做啊

**Comment by HzJayden** [comment]
> 我读了一下原文，感觉除了relu的问题还有一个就是文章中提到的coarse1 这一层网络代码里面是不是没有做啊

请问你知道他这个数据集是如何处理得到的depth.mat和image.mat吗？是如何把NYU_V2分开的呀
